From 953818c49b3cfa7d789ecfdf5575b92d0fdddba9 Mon Sep 17 00:00:00 2001
From: Brendan Le Foll <brendan.le.foll@intel.com>
Date: Tue, 8 Sep 2015 11:44:14 +0100
Subject: [PATCH] cmake: Change the install path of the Java bindings libraries

Signed-off-by: Brendan Le Foll <brendan.le.foll@intel.com>
---
 src/java/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/java/CMakeLists.txt b/src/java/CMakeLists.txt
index 34c348d..8c3558b 100644
--- a/src/java/CMakeLists.txt
+++ b/src/java/CMakeLists.txt
@@ -34,8 +34,8 @@ if (DOXYGEN_FOUND)
 endif ()
 
 install (FILES ${CMAKE_CURRENT_BINARY_DIR}/libmraajava.so
-		DESTINATION lib/java
+		DESTINATION lib/iotdk/i386
 )
 install (FILES ${CMAKE_CURRENT_BINARY_DIR}/mraa.jar
-		DESTINATION lib/java
+		DESTINATION lib/iotdk/ext
 )
-- 
2.5.1

