From 27870e0a101c6cfafdcbc9585129a791d6331489 Mon Sep 17 00:00:00 2001
From: Brendan Le Foll <brendan.le.foll@intel.com>
Date: Tue, 15 Sep 2015 10:19:23 +0100
Subject: [PATCH] cmake: revert to depend on mraa 0.7.5

Signed-off-by: Brendan Le Foll <brendan.le.foll@intel.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d81c863..6f85c34 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,7 +15,7 @@ find_package (PkgConfig REQUIRED)
 
 # force a libmraa search and minimum required version every time a config is generated
 unset(MRAA_FOUND CACHE)
-pkg_check_modules (MRAA REQUIRED mraa>=0.7.6)
+pkg_check_modules (MRAA REQUIRED mraa>=0.7.5)
 message (INFO " found mraa version: ${MRAA_VERSION}")
 
 # Appends the cmake/modules path to MAKE_MODULE_PATH variable.
-- 
2.5.0

